explorer/autoupdate_testdata.py

```diff
@@ -25,6 +25,16 @@ def main() -> None:
         "--tool=explorer",
         "--testdata=explorer/testdata",
     ] + sys.argv[1:]
+    subprocess.call(args)
+
+    args = [
+        str(autoupdate_py),
+        # Flags to configure for explorer testing.
+        "--tool=explorer",
+        "--testdata=explorer/testdata/trace",
+        "--autoupdate_arg=--trace_file=-",
+        "--autoupdate_arg=-trace_all",
+    ] + sys.argv[1:]
     exit(subprocess.call(args))
```

Comment:
Make sure that you aren't just churning your trace files by updating them with two different commands. These updates should be non-overlapping, much like `autoupdate_lit_testdata`.

Reply:
Thanks for letting me know about this!
I did some changes, please let me know if this works.

Reply:
It doesn't. When you run the script, note it says you can pass arguments: make sure that behavior works.

Probably this script shouldn't be modified to execute autoupdate_py twice, it should be once. Either use two separate scripts, or add flags to the base.

Reply:
I should add -- also, testdata is a str and not a list. Please make sure you test your changes before asking for review.

Reply:
Oh I'm sorry about that, thanks for reviewing! Will do these suggested changes.